spec/helpers/navigation_help_spec.rb

```diff
@@ -28,11 +28,11 @@ def it_does_not_render_for(path, level, depth = 1)
     stripped_html(actual) == stripped_html(expected)
   end
 
-  failure_message_for_should do |actual|
+  failure_message do |actual|
     "expected that\n#{actual}\nwould be equlivent to\n#{expected}"
   end
 
-  failure_message_for_should_not do |actual|
+  failure_message_when_negated do |actual|
     "expected that\n#{actual} would not be equlivent to\n#{expected}"
```

Comment:
I try. Going to swap the commits around so it's clear I didn't introduce the typo in the first commit :sunglasses: